pkg/datapath/linux/utime/cell.go

```diff
@@ -4,24 +4,15 @@
 package utime
 
 import (
-	"context"
-	"fmt"
 	"log/slog"
 
 	"github.com/cilium/hive/cell"
+	"github.com/cilium/hive/job"
 
-	"github.com/cilium/cilium/pkg/controller"
 	"github.com/cilium/cilium/pkg/maps/configmap"
 	"github.com/cilium/cilium/pkg/time"
 )
 
-const (
-	syncControllerName     = "sync-utime"
-	syncControllerInterval = 1 * time.Minute
```

Comment:
nit: I wouldn't mind keeping the `syncControllerInterval` constant, but YMMV

Reply:
i wanted to align it with most of the rest of the usages of hive timer jobs. they either use a config value or have it inlined. probably better to make this configurable in the future anyway. will keep it as is.